src/django/api/serializers/annotation.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework.serializers import ModelSerializer, Serializer
+from rest_framework.fields import CharField, BooleanField
+
+from ..models.submission import Annotation
+from ..fields import PointField
+
+
+class NewAnnotationSerializer(ModelSerializer):
+    location = PointField()
+
```

Comment:
Since there is nothing interesting happening here, let's drop this custom serializer field and just use the built-in.

```suggestion
```